src/retinanet.py

```diff
@@ -159,7 +159,7 @@ def __init__(self, num_classes=20):
         super().__init__()
         self.num_classes = num_classes
 
-    def focal_loss(self, x, y):
+    def focal_loss(self, x, y, labels):
```

Comment:
@apyskir I think that it would be better not to override original loss. With two separate classes we could switch between them and monitor improvement.

Reply:
@taraspiotr OK. I think I can just add another method in `RetinaLoss`, next to `focal_loss` and `focal_loss_alt`